swig/csharp/CMakeLists.txt

```diff
@@ -247,37 +651,4 @@ if (CSHARP_INTERPRETER OR WIN32)
     set(TEST_ENV "${NEW_TEST_ENV}")
   endif ()
 
-  add_test(NAME csharp_createdata COMMAND ${CSHARP_INTERPRETER} createdata.exe
```

Comment:
the tests should be added back

Reply:
Added the complete set of tests from the legacy scripts and tested and working on Win / Mac / Linux